pages/privacy/index.vue

```diff
@@ -1,26 +1,31 @@
 <template>
-  <div class="privacy-policy">
-    <div class="page-wrap container">
-      <div class="subpage">
-        <el-row type="flex" justify="center">
-          <el-col :row="24">
-            <p>
-              Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet
-              similique numquam accusamus quae, aliquid vero obcaecati facilis
-              reprehenderit. Fugit quibusdam placeat repellendus vitae similique
-              reprehenderit, aspernatur velit blanditiis non dicta.
-            </p>
-          </el-col>
-        </el-row>
-      </div>
-    </div>
-  </div>
+  <MarkdownPage :header="header" :body="body" />
```

Comment:
Components in [a template should be snake case](https://vuejs.org/v2/style-guide/#Component-name-casing-in-templates-strongly-recommended). This works, but we've adopted the "Everywhere" rule. I wish this was a separate rule so our linter could enforce this.
```suggestion
  <markdown-page :header="header" :body="body" />
```